apiary-metastore-events/apiary-hive-events/src/main/java/com/expediagroup/apiary/extensions/events/metastore/event/ApiaryListenerEventFactory.java

```diff
@@ -32,15 +31,15 @@
 
 public class ApiaryListenerEventFactory {
 
-  private final Configuration config;
+  private String hiveMetaStoreUris;
 
-  public ApiaryListenerEventFactory(Configuration config) {
-    this.config = config;
+  public ApiaryListenerEventFactory(String hiveMetaStoreUris) {
+    this.hiveMetaStoreUris = hiveMetaStoreUris != null ? hiveMetaStoreUris : "";
```

Comment:
do we need metastore uris in the configuration and event?
metastore uri can be different based on how client is connecting.
also sending uri in each event seems inefficent.
can we remove code related to metastore uri?

Reply:
Yep will remove this. Presuming we also don't need `HIVE_VERSION` sent in the event either so will remove this too, thanks

Reply:
`HIVE_VERSION` could be useful in case we change things here for a new version of Hive. The receiver can have customization for each version if not compatible.